addon/components/flash-message.js

```diff
@@ -5,11 +5,11 @@ const { computed, getWithDefault } = Ember;
 export default Ember.Component.extend({
   classNames        : [ 'flashMessage' ],
   classNameBindings : [ 'alertType' ],
-  messageStyle      : 'bootstrap',
+  messageStyle      : undefined,
 
   alertType: computed('flash.type', function() {
     const flashType    = getWithDefault(this, 'flash.type', '');
-    const messageStyle = getWithDefault(this, 'messageStyle', '');
+    const messageStyle = getWithDefault(this, 'messageStyle', Ember.ENV.flashTheme || 'bootstrap');
```

Comment:
Also actually, we should move this into the service instead. The flash message component and object don't need to know about defaults